framework/PostProcessors/ParetoFrontierPostProcessor.py

```diff
@@ -78,15 +93,25 @@ def _handleInput(self, paramInput):
     """
     costID  = paramInput.findFirst('costID')
     self.costID  = costID.value
+    self.invCost = costID.parameterValues['inv']
+
     valueID = paramInput.findFirst('valueID')
     self.valueID = valueID.value
+    self.invValue = valueID.parameterValues['inv']
 
     costLimit  = paramInput.findFirst('costLimit')
     if costLimit is not None:
       self.costLimit  = costLimit.value
+      self.costLimitType = costLimit.parameterValues['type']
+
     valueLimit = paramInput.findFirst('valueLimit')
     if valueLimit is not None:
       self.valueLimit = valueLimit.value
+      self.valueLimitType = valueLimit.parameterValues['type']
+
+    quadrant  = paramInput.findFirst('quadrant')
```

Comment:
quadrant is not mentioned in the manual and input specification

Reply:
this is an old version of the PP. It has been removed.